bin/users.ts

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
```

Comment:
Why is there both a bin/users.ts and bin/users? One of these need to go.

Reply:
users.ts was used to ensure users was strong typed, got a few warnings from it which is now fixed. I agree only one is necessary, but it was nice to ensure the file was strong typed i.e. i want to keep both... I suggest finding a workaround for this later (a lot of linting needs to be fixed for these files to run anyway), remove whichever becomes useless to us.